src/monitor/src/monitor/HostMonitorManager.cc

```diff
@@ -220,6 +221,14 @@ void HostMonitorManager::stop_host_monitor(int oid)
 /* -------------------------------------------------------------------------- */
 /* -------------------------------------------------------------------------- */
 
+void HostMonitorManager::raft_status(bool leader)
```

Comment:
better call this function is_leader and update class attribute _is_leader to follow convention

Reply:
This one conflicts with previous comment to call raft_status with string.
Done everything except this one. Waiting for Jenkins to test in microenv